esx_service/utils/auth_data.py

```diff
@@ -664,7 +663,10 @@ def new_db(self):
         err = self.__create_default_tenant()
         if err:
             return err
-        err = self.__create_default_privileges()
+        err = self.__create_all_ds_privileges_for_default_tenant()
+        if err:
+            return err
+        err = self.__create_vm_ds_privileges_for_default_tenant()
```

Comment:
I think new_db() should remove the new db is anything failed. 
it's a good catch and a quick fix. 
I suggest adding try...finally in new_db()
```python
def new_db)self):
  err= self.__connect()
   if err:
    return err
  try:
    <the rest of existing code>
  finally:
    if err:
        self.__close()
        os.remove(self.db_path)
```